tests/queries/0_stateless/03357_system_errors_query_id.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# Tags: no-parallel
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+query_id="01545_system_errors-$CLICKHOUSE_DATABASE"
+$CLICKHOUSE_CLIENT --query_id="$query_id" -q "select query_id;" &>/dev/null
+
+retry=0
+max_retries=30
+while [[ $($CLICKHOUSE_CLIENT -q "SELECT count(*) FROM system.errors WHERE query_id = '$query_id' and code = 47;") -ne 1 ]]; do
+  retry=$((retry+1))
+
+  if [ $retry -ge $max_retries ]; then
+    echo "Failed to query from system.error by query_id"
+    break
+  fi
+
+  sleep 0.5;
+done
+
+$CLICKHOUSE_CLIENT -q "SELECT count(*) FROM system.errors WHERE query_id = '$query_id' and code = 47;"
```

Comment:
```suggestion
$CLICKHOUSE_CLIENT -q "SELECT count(*) FROM system.errors WHERE query_id = '$query_id' and code = 47"
```

Reply:
The most important thing is that this should be gone, you already have a check above and this is racy, so let's simply replace it with `echo "OK"`